src/routes/github/configuration/github-configuration-post.ts

```diff
@@ -7,6 +8,20 @@ import { GitHubUserClient } from "~/src/github/client/github-user-client";
 import { GitHubAppClient } from "~/src/github/client/github-app-client";
 import { getCloudInstallationId } from "~/src/github/client/installation-id";
 
+const hasAdminAccess = async (gitHubInstallationId: number, githubToken: string, logger: Logger): Promise<boolean>  => {
+	const gitHubUserClient = new GitHubUserClient(githubToken, logger);
```

Comment:
The top level function has no need to be aware of these clients, so have moved them inside this function